### PR TITLE
Updated contribution policy

### PR DESCRIPTION
**Pull request by mmoreram**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,12 +46,14 @@ that your code fulfills these standards before any Pull Request.
 
 ``` bash
 composer.phar update
-bin/php-cs-fixer fix
-bin/php-formatter formatter:use:sort src/
-bin/php-formatter formatter:header:fix src/
+composer.phar check-style
```
Comment (EmanueleMinotto) [inline README.md]: If we consider composer as a system-wide script, here just `composer` should be used (preferred), if not I would suggest a `php composer.phar`.

File: README.md [hunk 2]
```diff
@@ -46,12 +46,14 @@ that your code fulfills these standards before any Pull Request.
...
 ```
 
 There is also a policy for contributing to this project. Pull requests must
 be explained step by step to make the review process easy in order to
 accept and merge them. New features must come paired with Unit and/or Functional
-tests.
+tests, check them using:
+
+``` bash
+composer.phar test
```
Comment (EmanueleMinotto) [inline README.md]: same as above

**Comment by EmanueleMinotto** [comment]
Also the `README.md` should be updated (in the Contributing section).

**Comment by mmoreram** [comment]
@EmanueleMinotto updated with your tips. I've also updated all other components and bundles contributing files.

**Comment by EmanueleMinotto** [comment]
great! :+1:

**Pull request merged**